lib/ansible/modules/cloud/azure/azure_rm_virtualmachine_scaleset.py

```diff
@@ -739,6 +746,8 @@ def exec_module(self, **kwargs):
                             ))
                         vmss_resource.virtual_machine_profile.storage_profile.data_disks = data_disks
 
+                    if image_reference is not None:
+                        vmss_resource.virtual_machine_profile.storage_profile.image_reference = image_reference
```

Comment:
when idempotent updating, should vmss_resource.virtual_machine_profile.storage_profile.image_reference keeps none  or  set as existing value?